CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Fixed 🐞
 
 -   Fix an issue with web demo on Safari showing a white screen and not loading [#3396](https://github.com/MaibornWolff/codecharta/pull/3396)
+-   Fix ccsh not finding parsers when run without flags [#3401](https://github.com/MaibornWolff/codecharta/pull/3401)
```

Comment:
I suggest to not have this changelog line. Let's instead add one line for the feature that has been added recently in commit 866544872eb17ba41343e0eaa8eaf3206e89c5e6 (#3387) in section `### Added`. The changelog entry for the feature itself has not been added yet.

By that we can remove the "fixed" line since it has not been released yet.